### PR TITLE
fix tab contents and labeling within groups.

### PR DESCRIPTION
**Pull request by pabender**
Still need to fix the alignment of the descriptions on the panel containing the group tabs, and possibly make the tabs wrap, but there is now only one tab with each label, and each tab has the appropriate contents.

### CODE REVIEW

**Comment by balazsracz** [comment]
Tested, looks much better, thanks.

There is still a problem with some inside panel's left alignment. My JMRI gets into an infinite loop of layout when displaying the tower-lcc (B4 firmware) CDI. I'll try to trace it down.

**Comment by balazsracz** [comment]
https://github.com/openlcb/OpenLCB_Java/pull/29 to fix the alignment

File: src/org/openlcb/cdi/swing/CdiPanel.java [hunk 1]
```diff
@@ -250,7 +251,7 @@ DisplayPane createGroupPane(CdiRep.Group item, long origin, int space) {
             }
             JPanel currentPane = this;
             JTabbedPane tabbedPane = new JTabbedPane();
-            currentPane.add(tabbedPane);
+            tabbedPane.setTabLayoutPolicy(JTabbedPane.SCROLL_TAB_LAYOUT);
```
Comment (balazsracz) [inline src/org/openlcb/cdi/swing/CdiPanel.java]: I actually liked the wrapping tabs better; the scroll makes it painfully slow to get to a large count on a small screen.

Comment (pabender) [inline src/org/openlcb/cdi/swing/CdiPanel.java]: That was part of an attempt to fix the scrolling alignment issue, and it didn't work.  I'm ok taking that out.

**Pull request merged**